spec/models/submission_packets_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
```

Comment:
Single quotes here, but double on line 4. Any reason Hound isn't enabled on this repo?